lib/veritrans/midtrans_error.rb

```diff
@@ -0,0 +1,10 @@
+class MidtransError < StandardError
+  attr_reader :status
+  attr_reader :data
+
+  def initialize(message, status_code = nil, api_response = nil)
+    super(message)
+    @status = status_code
+    @data = api_response
+  end
```

Comment:
As previously reminded, please mind these points:

### Descriptive var/attribute names:
`status` is not very descriptive, what status do you mean: payment? API? look at the [example](https://github.com/Midtrans/midtrans-nodejs-client#3-handling-error--exception), should be something like `http status code`.

`data` what data do you even mean?

### Your implementation should be at least on same standard or better than the example/reference:
The reference have proper descriptive attributes name, your implementation is slightly worse. Why your implementation don't include the `rawHttpClientData` ? Please add, the name can also improved to be more descriptive to `raw http client response`